docs/reference/configuration/ingress.md

```diff
@@ -132,3 +132,68 @@ packages:
             - path: tls.credentialName
               value: tenant-gateway-tls-secret # Reference to the Kubernetes secret for the tenant gateway's TLS certificate
 ```
+
+### Apex (Root) Domain Configuration
+By default, the UDS Core Gateways are configured with wildcard hosts (for example, `*.uds.dev`), which match only subdomains (such as `demo.uds.dev` or `keycloak.admin.uds.dev`). The apex domain (i.e. `uds.dev`) is not covered by a wildcard. This is important if you need an application to be accessible at the root of your domain.
+
+To support this use case, UDS Core provides an optional configuration to enable a dedicated server block for the apex domain. When enabled, two additional server blocks are added to your Istio Gateway:
+- **HTTP on port 80**: Redirects traffic to HTTPS.
+- **HTTPS on port 443**: Terminates TLS using settings from the rootDomainTLS section.
+
+If you want your application to be reachable at `https://uds.dev`, enable apex (root) domain configuration via a bundle override in your UDS Bundle. For example:
+```yaml
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.23.0-upstream
+    overrides:
+      istio-tenant-gateway:
+        uds-istio-config:
+          values:
+            - path: enableRootDomain
+              value: true
+            - path: rootDomainTLS.mode
+              value: SIMPLE
+            - path: rootDomainTLS.credentialName
+              value: ""  # Leave blank to auto-create the secret using the provided cert data.
+            - path: rootDomainTLS.supportTLSV1_2
+              value: true
+            - path: rootDomainTLS.cert
+              value: "BASE64_ENCODED_CERTIFICATE"
+            - path: rootDomainTLS.key
+              value: "BASE64_ENCODED_PRIVATE_KEY"
+            - path: rootDomainTLS.cacert
+              value: "BASE64_ENCODED_CERTIFICATE"
```

Comment:
nit: these would/should normally be variables rather than values (see other example above for the wildcard)